src/Cms/PageActions.php

```diff
@@ -329,7 +329,7 @@ protected function commit(string $action, array $arguments, Closure $callback)
         if ($action === 'create') {
             $argumentsAfter = ['page' => $result];
         } elseif ($action === 'duplicate') {
-            $argumentsAfter = ['duplicatePage' => $result];
+            $argumentsAfter = ['duplicatePage' => $result, 'originalPage' => $old];
```

Comment:
I have to add this line to pass unit test.

Reply:
@afbora I have rebased your PR branch against mine, which automatically updated your commit so that it doesn't need the change anymore. Advantage: Once my PR is merged, yours can be merged as well, otherwise there would potentially be a merge conflict.

I'm marking this PR as blocked as it now depends on #2706.